freeipa/src/main/resources/application-dev.yml

```diff
@@ -0,0 +1 @@
+cb.image.catalog.url: https://cloudbreak-imagecatalog.s3.amazonaws.com/freeipa-image-catalog.json
```

Comment:
This is already there in: https://github.com/hortonworks/cloudbreak/blob/master/freeipa/src/main/resources/application.yml#L512